src/Client/index.ts

```diff
@@ -88,6 +89,7 @@ export class Client {
       return new Client(
         {
           domain: address,
+          port: port,
```

Comment:
The default port for connection strings is `2113` in connectionString. Is this correct, or do we want to change it / enforce it for `+discover`?

Reply:
Yes but it the previous design, it forced the user to only operate on the 2113. Now, if the user asks for a diiferent port number for discovery, it is threaded down to where we do the DNS discovery.